### ISSUE TITLE
[bug] redis_db option is not used

### ISSUE THREAD

**Issue by capricornusx** [centrifugal/centrifugo#487]
Centrifugo version is **3.0.3** (in Docker)
Operating system is **Ubuntu 20.10**

`config.json:`
```json
{
"address": "127.0.0.1",
"port": 8002,

"engine": "redis",
"redis_prefix": "centrifugo",
"redis_tls_skip_verify": true,
"redis_address": ["localhost:6379"],
"redis_db": "3",
"redis_tls": false,
"redis_prefix": "centrifugo",
"history_meta_ttl": "86400s",

"log_level": "debug",
"token_hmac_secret_key": "XXX",
"admin_password": "YYY",
"admin_secret": "ZZZ",
"admin": true,
"api_key": "XXX",
"prometheus": true,
"health": true,
"client_user_connection_limit": 1,
"websocket_message_size_limit": 1000000,
"websocket_compression": true,
"namespaces": [
	{
	"name": "public",
	"anonymous": false,
	"publish": true,
	"watch": true,
	"presence": true,
	"join_leave": false,
	"history_size": 10,
	"sockjs_heartbeat_delay": 30,
	"recover": true,
	"history_ttl": "30s"
	}
],
"allowed_origins": ["*"],
"proxy_http_headers": [
  "Origin",
  "User-Agent",
  "Cookie",
  "Authorization",
  "X-Real-Ip",
  "X-Forwarded-For",
  "X-Request-Id"
]
}
```

`docker-compose logs -f centrifugo:`

```log
centrifugo  | {"level":"info","version":"3.0.3","runtime":"go1.17.2","pid":1,"engine":"Redis","gomaxprocs":12,"time":"2021-10-15T15:35:48Z","message":"starting Centrifugo"}
centrifugo  | {"level":"info","path":"/centrifugo/config.json","time":"2021-10-15T15:35:48Z","message":"using config file"}
centrifugo  | {"level":"info","algorithms":"HS256, HS384, HS512","time":"2021-10-15T15:35:48Z","message":"enabled JWT verifiers"}
centrifugo  | {"level":"info","time":"2021-10-15T15:35:48Z","message":"Redis: localhost:6379/0, using password: false"}
centrifugo  | {"level":"debug","shard":"localhost:6379","time":"2021-10-15T15:35:48Z","message":"running Redis control PUB/SUB, num workers: 12"}
centrifugo  | {"level":"debug","shard":"localhost:6379","time":"2021-10-15T15:35:48Z","message":"running Redis PUB/SUB, num workers: 12"}
centrifugo  | {"level":"debug","shard":"localhost:6379","time":"2021-10-15T15:35:48Z","message":"starting RedisBroker Subscriber"}
centrifugo  | {"level":"warn","time":"2021-10-15T15:35:48Z","message":"usage of allowed_origins * is discouraged for security reasons, consider setting exact list of origins"}
centrifugo  | {"level":"info","time":"2021-10-15T15:35:48Z","message":"serving websocket, API, admin, prometheus, health endpoints on 127.0.0.1:8002"}
```

`👉 Redis: localhost:6379/0` 

but I'am use `"redis_db": "3"`

**Comment by FZambia** [comment]
@capricornusx hello, thanks - fixed in 633d2ce73243542cf06d28bce863ae9db2a51acc, will be part of v3.0.4

**Comment by FZambia** [comment]
v3.0.4 released.
